fluster/main.py

```diff
@@ -47,23 +49,57 @@ class Main:
     def __init__(self) -> None:
         self.decoders_dir = DECODERS_DIR
         self.test_suites_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), TEST_SUITES_DIR
-        )
-        # Only use the system directory for test suites if the local directory
-        # doesn't exist and the system directory does exist.
-        if (
-            sys.platform.startswith("linux")
-            and not os.path.exists(self.test_suites_dir)
-            and os.path.exists(TEST_SUITES_DIR_SYS)
-        ):
-            self.test_suites_dir = TEST_SUITES_DIR_SYS
-
+            os.path.dirname(__file__), "..", TEST_SUITES_DIR
+        )
         self.resources_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESOURCES_DIR
+            os.path.dirname(__file__), "..", RESOURCES_DIR
         )
-        self.results_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESULTS_DIR
+        self.results_dir = os.path.join(gettempdir(), RESULTS_DIR)
+
+        is_uninstalled = os.path.exists(
+            os.path.join(os.path.dirname(__file__), "..", ".git")
         )
+        if not is_uninstalled:
+            appname = "fluster"
+            site_data_dir = platformdirs.site_data_dir(appname)
+            user_data_dir = platformdirs.user_data_dir(appname)
+
+            test_suites_dir = os.path.join(site_data_dir, TEST_SUITES_DIR)
+            if os.path.exists(test_suites_dir):
+                self.test_suites_dir = test_suites_dir
+            else:
+                self.test_suites_dir = os.path.join(user_data_dir, TEST_SUITES_DIR)
+
+            resources_dir = os.path.join(site_data_dir, RESOURCES_DIR)
```

Comment:
The resources directory is the location where test vectors are downloaded and therefore it can't be a system path: packages will never ship test vectors.

Reply:
I was thinking in a use case like a volume mounted in a container that executes the tests where the test vectors are pre-installed.

Reply:
The problem with this approach is that the default location changes depending on whether a directory exists. You don't want that behaviour for a default data directory, unless you do it for backward compatibility, as we do for the uninstalled directories.
The use case you are describing can be done by passing `-r` or mounting the volume in `~/.local/share/fluster/resources`

Reply:
Done in f409dcb